src/app/config.rs

```diff
@@ -705,6 +727,38 @@ fn parse_dns_suffix(s: &str) -> Result<dns::Suffix, ParseError> {
         .map_err(|_| ParseError::NotADomainSuffix)
 }
 
+pub fn parse_backoff<S: Strings>(
+    strings: &S,
+    base: &str,
+    default: Backoff,
+) -> Result<Backoff, Error> {
+    let env_disabled = format!("LINKERD2_PROXY_{}_EXP_BACKOFF_DISABLED", base);
+    let disabled = strings
+        .get(&env_disabled)?
+        .map(|d| !d.is_empty())
+        .unwrap_or(false);
+
+    if disabled {
+        Ok(Backoff::None)
+    } else {
+        let min_env = format!("LINKERD2_PROXY_{}_EXP_BACKOFF_MIN", base);
+        let min = parse(strings, &min_env, parse_duration);
+        let max_env = format!("LINKERD2_PROXY_{}_EXP_BACKOFF_MAX", base);
+        let max = parse(strings, &max_env, parse_duration);
+        let jitter_env = format!("LINKERD2_PROXY_{}_EXP_BACKOFF_JITTER", base);
+        let jitter = parse(strings, &jitter_env, parse_number::<f64>);
```

Comment:
While it'd be a bad idea, this could parse a negative jitter. Perhaps this should require positive jitter?